### DISCUSSION TITLE
Are you taking pull requests?

### DISCUSSION THREAD

**Discussion by omidantilong** [RudeySH/lastfm-bulk-edit#27] [General]
Hey @RudeySH , thanks again for your hard work on this script. Just wondering if you're open to PRs? I've got a couple of very small changes on a local branch. The first one just deals with tightening the styling of the new album name display. 

Let me know, I would be glad to contribute to the ongoing work.

Cheers!

**Comment by RudeySH** [comment]
Yes, feel free to contribute!
